testsuite/tests/kuadrant/gateway/conftest.py

```diff
@@ -44,12 +44,12 @@ def exposer(request, hub_openshift) -> Exposer:
 
 
 @pytest.fixture(scope="session")
-def cluster_issuer():
+def cluster_issuer(testconfig):
     """Reference to cluster self-signed certificate issuer"""
     return CustomReference(
```

Comment:
don't you also need to validate dynaconf here?

Reply:
`control_plane` is validated by default